### ISSUE TITLE
Application version number is wrong

### ISSUE THREAD

**Issue by dynamicsblog** [MicrosoftDocs/dynamics365smb-devitpro-pb#868]
after the upgrade Support and Help page and Get-NAVApplic.. cmdlet shows wrong version number. ideas? 
https://www.yammer.com/dynamicsnavdev/#/Threads/show?threadId=412306527985664&search_origin=global&scoring=linear1Y-prankie-group-private-higher&match=any-exact&search_sort=relevance&page=1&search=can%20someone%20tell%20me

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5e435e27-0b74-7dcd-555d-977517fd3774
* Version Independent ID: ea922161-12f9-3f86-69f9-2044369d6574
* Content: [Upgrade an unmodified application - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/upgrade/upgrade-unmodified-application#feedback)
* Content Source: [dev-itpro/upgrade/upgrade-unmodified-application.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/live/dev-itpro/upgrade/upgrade-unmodified-application.md)
* Service: **dynamics365-business-central**
* GitHub Login: @jswymer
* Microsoft Alias: **jswymer**

**Comment by jswymer** [comment]
Hello,
The Application Version shown on the Help and Support page and by the Get-NavApplication cmdlet comes from the $ndo$dbproperty table in the database. This version is not automatically updated during upgrade or by the Invoke-NAVpplicationDatabaseConversion cmdlet; it can only be boosted manually by using the Set-NavApplication cmdlet. This is a result of the move to a fully extension-based solution. For now, the best way to get the application version is looking at the version of the base application extension. I hope this helps. Thanks for the feedback!

**Comment by dynamicsblog** [comment]
Hi, thank you for sharing this. Then it seems like that the cmdlet didnt boost that actually. I have repeated the steps more than one time, and always teh application version no. was wrong after the update. We will set this manually. Thank you!

**Comment by staedter** [comment]
We have the same Problem. I also tried already to set the Application Version manually via the Powershell Cmdlet, but after that Get-NavTenant shows the Database is in State 'OperationalDataUpgradePending' ...

I already tried (after Snapshotting the VM) to start-navdataupgrade again but it fails with this error: 

Start-NAVDataUpgrade : Cannot start the data upgrade process because tenant version is already the latest application version the tenant database supports (14.3.34444.0). Call Sync-NAVTenantDatabase to sync the tenant database to the
current version of the application (15.1.37881.38071).

Sync-NavTenantDatabase then fails, because it is not a multitenant environment...

I am actually at a loss right now. Any suggestion would be greatly appreciated!
